bin/cylc-suite-state

```diff
@@ -131,6 +151,43 @@ suite_poller.add_to_cmd_options(parser)
 
 suite = args[0]
 
+if options.use_task_point and options.cycle:
+    sys.exit( "ERROR: cannot specify a cycle point and use environment variable" )
```

Comment:
Style violations!  (space after parenthesis) ... but the entire file is stylistically "transitional", so I'll gloss over it.